bitnami/grafana-loki/values.yaml

```diff
@@ -164,6 +172,27 @@ loki:
           addresses: dns+{{ include "grafana-loki.memcached-index-queries.host" . }}
           service: http
         {{- end }}
+      # New tsdb-shipper configuration
```

Comment:
Please, remove the comments in this configuration file, at least those not strictly necessary. They make sense in [TSDB: Example Configuration](https://grafana.com/docs/loki/latest/operations/storage/tsdb/#example-configuration) for illustrative purposes but I think they are not necessary here.

Reply:
Done

Reply:
Thank you for addressing this.